src/nrepl/index.ts

```diff
@@ -829,7 +829,13 @@ export class NReplEvaluation {
                     this.session.stacktrace().then((stacktrace) => {
                         this._stacktrace = stacktrace;
                         this.doReject(this.exception);
-                    }).catch(() => { });
+                    }).catch((e) => {
+                        // This failure occurs  when the `stacktrace` cider-nrepl
+                        // middleware is not available. In this case we can still
```

Comment:
Did you mean to say "still print/show/return the error message"?

Reply:
yup, looks like I dropped that word when formatting the comment to wrap nicely.